code/modules/jobs/jobs.dm

```diff
@@ -45,7 +45,10 @@ GLOBAL_LIST_INIT(civilian_positions, list(
 	"Chaplain",
 	"Clown",
 	"Mime",
-	"Assistant"))
+	"Assistant",
+	"Gimmick",
+	"Barber",
+	"Stage Magician"))

```

Comment:
Couldn't you just have `/datum/job/gimmick/New()` add the job's title to this list?

See other comment about reducing the amount of places in the code that need to be edited to add gimmick jobs.

Reply:
done